### PR TITLE
Add ru translations for readme and tutorial

### PR DESCRIPTION
**Pull request by Zuzuk-null**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ Swarm
 [![Build Status](https://github.com/byorgey/swarm/actions/workflows/haskell-ci.yml/badge.svg)](https://github.com/byorgey/swarm/actions)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
 ![GitHub Contributors](https://contrib.rocks/image?repo=byorgey/swarm)
-
+--
+Other langs: [ru](other_langs/README_RU.md)
```
[the remark this answers is not in the document]
Comment (xsebek) [reply README.md:8]: I think this would look better or we could add bigger flag images to repo.

**Comment by Zuzuk-null** [comment]
> Thanks @Zuzuk-null , this is wonderful! One question I have: these files (especially the tutorial) will probably still change a lot. Would you be willing to keep the Russian translations up to date when they change? (Happy to @ mention you when that happens so you don't have to be constantly watching to see if they change.)

Yes

**Comment by byorgey** [comment]
> Yes

Great!  Well, the tutorial has already changed since you opened this PR... could you update it please? =)

File: other_langs/README_RU.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+--
+Other langs: [us🇺🇸](../README.md)
+--
+Swarm - это 2D игра про программирование и сбор ресурсов.
+Программируйте своих роботов, чтобы исследовать мир и собирать
+ресурсы, что, в свою очередь, позволяет вам создавать
+модернизированных роботов, которые могут запускать более
+интересные и сложные программы.
+
+![](images/log.png)
```
[the remark this answers is not in the document]
Comment (xsebek) [reply other_langs/README_RU.md:16]: @Zuzuk-null you still have the image missing :wink: If my google-translate Russian is correct, you can commit it with the button above :slightly_smiling_face:

**Comment by byorgey** [comment]
Just looking through some old PRs and noticed this one is still open.  @Zuzuk-null , thanks for updating the translation again. But it looks like there were also some comments/suggestions from @xsebek that you never responded to.  If you address those I will merge this.

**Pull request closed without merging**